sdk/template/template/src/tracing.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getTracer } from "@azure/core-tracing";
```

Comment:
replace this whole file with a call to use `createSpanFunction`: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-http/src/createSpan.ts

Reply:
Cool!